superset/viz.py

```diff
@@ -1231,6 +1232,39 @@ def get_data(self, df):
         return df.to_dict(orient='records')
 
 
+class ChordViz(BaseViz):
```

Comment:
This is the backend part of this component, you can derive `BaseViz` or other `BaseViz` derivatives